.evergreen/evergreen.yml.in

```diff
@@ -121,6 +121,24 @@ const EXECUTABLE_PKG_INFO = [
 %>
 exec_timeout_secs: 3600
 
+post_error_fails_task: true
+post:
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |
+        mkdir -p npm-logs
+        tar cvzf npm-logs.tgz npm-logs
+  - command: s3.put
+    params:
+      aws_key: ${aws_key}
+      aws_secret: ${aws_secret}
+      local_file: npm-logs.tgz
+      remote_file: mongosh/binaries/${revision}/${revision_order_id}/npm-logs-${build_variant}-${task_name}-${task_id}.tgz
+      bucket: mciuploads
+      permissions: public-read
+      content_type: application/x-gzip
```

Comment:
Uploading all npm logs from a run was helpful for debugging, I’d just keep it here.